SubEthaEdit-Mac/Source/PlainTextDocument.m

```diff
@@ -2390,7 +2390,20 @@ - (void)TCM_ensureFileTypeDataOrSEEText {
 - (void)saveDocumentWithDelegate:(id)delegate didSaveSelector:(SEL)didSaveSelector contextInfo:(void *)contextInfo {
     if ([self TCM_validateDocument]) {
 		[self TCM_ensureFileTypeDataOrSEEText];
-        [super saveDocumentWithDelegate:delegate didSaveSelector:didSaveSelector contextInfo:contextInfo];
+        
+        // If the file exists TCM_validateDocument handles the case whether a file
+        // has been modified by another application, so the super method would display
+        // a misleading dialog.
+        if([NSFileManager.defaultManager fileExistsAtPath:self.fileURL.path]) {
```

Comment:
Not sure if this redirection is correct this way. (Also we should have called the didSaveSelector even on cancellation in the past too if I read the docs right). Need to look at the behaviour you like to circumvent. 

Also style: please space after `if` and before `(`